examples/temporal_graph_learning/README.md

```diff
@@ -0,0 +1,82 @@
+# Customized TGAT model for snapshot-based temporal graph learning
+
+This folder contains Python codes that demonstrate how to leverage the GraphStorm Framework (GSF) for production-level snapshot-based temporal graph data. This example utilizes the TGAT as a GNN model and the MAG graph data.
+
+## How to run this example
+
+The following commands assume the user is in the directory `graphstorm/examples/temporal_graph_learning/`.
+
+**Step 1: Prepare the MAG dataset for using the GraphStorm.**
+
+Download dataset
+```
+mkdir ./DATA
+wget -P ./DATA/GDELT https://s3.us-west-2.amazonaws.com/dgl-data/dataset/tgl/GDELT/node_features.pt
+wget -P ./DATA/GDELT https://s3.us-west-2.amazonaws.com/dgl-data/dataset/tgl/GDELT/labels.csv
+wget -P ./DATA/GDELT https://s3.us-west-2.amazonaws.com/dgl-data/dataset/tgl/GDELT/edges.csv
+```
+
+**Step 2: Pre-processing the raw data into GraphStorm/DGL's data structure**
+
+Run the command to convert the data into the required raw format:
+```
+python3 gen_graph.py
+```
+
+Once successful, the command will generate a set of folders and files under the `./DATA` folder, which contains pre-processed graph data recognizable by the GraphStorm's graph construction tool.
```

Comment:
Can we explain what the data is split by time and what the outcomes of the `gen_graph.py` script? 

Because temporal information is the key concept in this example, it is better to have some explanation about how we construct data to fit the TGAT. thanks

Reply:
I made some updates to README.md please check. Please let me know if need to include more information.